src/uct/sm/scopy/knem/knem_iface.h

```diff
@@ -23,4 +23,5 @@ typedef struct uct_knem_iface {
     uct_knem_md_t                 *knem_md;
 } uct_knem_iface_t;
 
+
```

Comment:
yes, since you are reviewing the last commit, it has the change with the previous ones - adding a new empty line in this file
but if you will review the entire PR, it should not have any changes in this file :)